unified_targets/configs/TALONF7FUSION.config

```diff
@@ -0,0 +1,116 @@
+# Betaflight / STM32F7X2 (S7X2) 
+board_name TALONF7FUSION
+manufacturer_id TLON
```

Comment:
This is not an official manufacturer id. Please follow the instructions at https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/CreatingAUnifiedTarget.md

Reply:
@mikeller  do I need to change anything here to make it good for merge?
thanks

Reply:
Yes, the `manufacturer_id` needs to be one listed in https://github.com/betaflight/betaflight/blob/master/unified_targets/docs/Manufacturers.md.

Reply:
@mikeller 
Updated the changes. thanks.